### PR TITLE
Add US Election Badging

### PR DESCRIPTION
**Pull request by sammorrisdesign-zz**
Some badging for the us election. Tried to follow the UK badge example as much as possible (which has since been removed).

screen shot 2016-01-28 at 12 03 20

screen shot 2016-01-28 at 12 03 29

### CODE REVIEW

File: common/app/views/fragments/meta/metaInline.scala.html [hunk 1]
```diff
@@ -2,7 +2,16 @@
...
 
-    <div class="content__labels">
+@defining(
+    item.content.tags.tags.exists(_.id == "us-news/us-elections-2016")
+){ isUSElection =>
+    <div class="content__labels @if(isUSElection) {content__labels--us-election}">
+
+        @if(isUSElection) {
+            <div class="badge-slot badge-slot--us-election">
+                <img class="badge-slot__img" src="http://static.guim.co.uk/sys-images/Guardian/Pix/pictures/2016/1/28/1453981123556/uselectionlogo.png"/>
```
Comment (jbreckmckye) [inline common/app/views/fragments/meta/metaInline.scala.html]: Is there a helper to inject the static path?

**Comment by rich-nguyen** [comment]
your screenshot is a eurozone badge?

File: static/src/stylesheets/module/content/_badging.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
+.badge-slot--us-election {
+    width: 33px;
+    margin-top: 2px;
```
Comment (jbreckmckye) [inline static/src/stylesheets/module/content/_badging.scss]: What's the 2px bump for?

Comment (sammorrisdesign) [inline static/src/stylesheets/module/content/_badging.scss]: X-height in our font

**Comment by sammorrisdesign** [comment]
@rich-nguyen was just for testing on liveblog background.

File: common/app/views/fragments/meta/metaInline.scala.html [hunk 3]
```diff
@@ -2,7 +2,16 @@
 
 @import common.{LinkTo, Localisation}
 
-    <div class="content__labels">
+@defining(
+    item.content.tags.tags.exists(_.id == "us-news/us-elections-2016")
+){ isUSElection =>
+    <div class="content__labels @if(isUSElection) {content__labels--us-election}">
```
Comment (rich-nguyen) [inline common/app/views/fragments/meta/metaInline.scala.html]: it's bit naff that we have to pepper our generic meta system with this kind of stuff. Although I accept that we just have to do this kind of thing. At the very least, can we wrap it around a switch? Then we can tidy later.

Comment (sammorrisdesign) [inline common/app/views/fragments/meta/metaInline.scala.html]: Ok I've added the switch, we cool?

File: common/app/conf/switches/FeatureSwitches.scala [hunk 4]
```diff
@@ -366,4 +366,13 @@ trait FeatureSwitches {
     sellByDate = never,
     exposeClientSide = true
   )
+
+  val USElectionSwitch = Switch(
+    "Feature",
+    "us-election",
+    "When ON, items tagged with us-news/us-elections-2016 will have visual elements added",
+    safeState = On,
+    sellByDate = never,
```
Comment (rich-nguyen) [inline common/app/conf/switches/FeatureSwitches.scala]: It shouldn't be `never`

Comment (sammorrisdesign) [inline common/app/conf/switches/FeatureSwitches.scala]: Just copied the one above :/

**Comment by rich-nguyen** [comment]
:+1:

**Pull request merged**